runtime/plugin/lsp.vim

```diff
@@ -0,0 +1,28 @@
+" TODO(tjdevries): Move this to autoload?
+
+execute('luafile ' . expand('<sfile>:h') . '/../lua/lsp/plugin.lua')
+
+function! LSP_Start() abort
```

Comment:
@tjdevries hi, I think maybe some people like autoload function more. for example 
```
function! lsp#start() abort
```